### ISSUE TITLE
Get array value does not work on version 1.0.0

### ISSUE THREAD

**Issue by luiscoms** [amenezes/config-client#45]
### Platform

All platforms

### Steps to Reproduce

After 1.0.0, `client.get` method does not retrieve array values.

Setting a config like this

```yml
example:
    - 1
    - 2
```

When I run `config.get('example')` #

### Expected Result

Prior 1.0.0

Results were

```python
> client.get('example')
[1, 2]
```

### Actual Result

After 1.0.0

Results are empty string

```python
> client.get('example')
''
```

#### PS: 

Running `client.get('example[0]')` return first value, but It's not useful because I don't know how many values are configured

```python
> client.get('example[0]')
1
```

**Comment by amenezes** [comment]
@luiscoms,

First thanks for report this bug. A new release - **1.0.2** - was published with the fix.

release: https://github.com/amenezes/config-client/releases/tag/1.0.2
pypi package: https://pypi.org/project/config-client/

**Comment by luiscoms** [comment]
Hi, unfurtunelly error still occurs, I have create some tests in order to help to find the fix on Pull Request #46
